test/cpp/end2end/xds_end2end_test.cc

```diff
@@ -765,6 +767,18 @@ class AdsServiceImpl : public AggregatedDiscoveryService::Service,
     resource_types_to_ignore_.emplace(type_url);
   }
 
+  void UnsetResource(const std::string& type_url, const std::string& name) {
+    grpc_core::MutexLock lock(&ads_mu_);
+    ResourceState& state = resource_map_[type_url][name];
+    ++state.version;
```

Comment:
unset is also a bump up in version?

Reply:
Yup.  Otherwise, the logic in `StreamAggregateResources()` won't send an update when a resource is deleted.